src/server_manager/ui_components/outline-share-dialog.html

```diff
@@ -103,25 +108,28 @@ <h3>[[localize('share-title')]]</h3>
       <div contenteditable id="selectableText" style="-webkit-text-size-adjust: 100%;">
         <p>[[localize('share-invite')]]</p>
 
-        <p><a href$="{{s3Url}}">{{s3Url}}</a></p>
+        <p><a contenteditable="false" href$="{{s3Url}}">{{s3Url}}</a></p>
```

Comment:
Yes, the outer div is `contenteditable` which seems to disable click events, so I disabled that for the link and now it can be clicked.

You can still select pieces of the div, but NOT the link. You can copy the whole link or none of it. Do you know of a better solution?

Reply:
Copying the link is important, but not copying parts of it. If we can copy the full link without the rest of the text, this is fine.

Reply:
👍